### ISSUE TITLE
Internal server error on macOS

### ISSUE THREAD

**Issue by fxcoudert** [xxczaki/cash-cli#20]
While testing version 2.1.0 of cash-cli for Homebrew (https://github.com/Homebrew/homebrew-core/pull/34565), we are seeing an `Internal server error` on a very simple operation:

```
==> /usr/local/Cellar/cash-cli/2.1.0/bin/cash 100 INR USD GBP
- Converting...
✖ Internal server error :(

Error: cash-cli: failed
```

The same error was already occurring with version 2.0.8.

**Comment by xxczaki** [comment]
Fixed in 2.1.1. You will now need to provide your own API key from https://fixer.io :cry:

**Comment by xxczaki** [comment]
@fxcoudert Please see #17
